test/ask/survey_log_entry_test.exs

```diff
@@ -19,4 +19,27 @@ defmodule Ask.SurveyLogEntryTest do
     changeset = SurveyLogEntry.changeset(%SurveyLogEntry{}, @valid_attrs)
     assert changeset.valid?
   end
+
+  test "changeset normalizes sms mode" do
+    changes = %{@valid_attrs | mode: "sms"}
+    changeset = SurveyLogEntry.changeset(%SurveyLogEntry{}, changes)
+    assert changeset.changes.mode == "SMS"
+  end
+
+  test "changeset normalizes ivr mode" do
+    changes = %{@valid_attrs | mode: "Ivr"}
+    changeset = SurveyLogEntry.changeset(%SurveyLogEntry{}, changes)
+    assert changeset.changes.mode == "IVR"
+  end
+
+  test "changeset normalizes mobileweb mode" do
+    changes = %{@valid_attrs | mode: "mobileWEB"}
+    changeset = SurveyLogEntry.changeset(%SurveyLogEntry{}, changes)
+    assert changeset.changes.mode == "Mobile Web"
+  end
+
+  test "changeset leaves unrecognized mode as it is" do
+    changeset = SurveyLogEntry.changeset(%SurveyLogEntry{}, @valid_attrs)
+    assert changeset.changes.mode == "some content"
```

Comment:
It's a bit weird to have `"some content"` as the mode for a `@valid_attrs` given that it isn't _that_ valid, I guess.

But not something to fix now.